Extension/i18n/chs/package.i18n.json

```diff
@@ -17,6 +17,7 @@
 	"c_cpp.command.configurationEditJSON.title": "编辑配置(JSON)",
```

Comment:
Please don't modify  i18n.json files directory.  These are output files form the location process and will get stomped on.  It's not necessary to update them when adding strings.  If a string if missing, it will fall back to English automatically.  (Even if you're trying to preserve a localized string when just renaming the ID, it will still get stomped on with an English version).

Reply:
Okay, yeah, I wasn't sure about that.